tests/stub/routing/test_routing_v4x3.py

```diff
@@ -477,7 +546,29 @@ def work(tx):
         self.assertEqual([[]], sequences)
         self.assertEqual(2, num_retries)
 
-    def test_should_retry_write_until_success_with_leader_shutdown_during_tx_using_tx_function(self):
+    @driver_feature(types.Feature.OPT_PULL_PIPELINING)
+    def test_should_retry_write_until_success_with_leader_change_using_tx_function(
+            self):
+        # TODO remove this block once all languages work
+        if get_driver_name() in ['python']:
+            self.skipTest("requires investigation")
+        self._should_retry_write_until_success_with_leader_change_using_tx_function(
+            leader_switch_script=
+            "writer_tx_with_unexpected_interruption_on_pipelined_pull.script"
+        )
+
+    def test_should_retry_write_until_success_with_leader_change_on_run_using_tx_function(
+            self):
+        # TODO remove this block once all languages work
+        if get_driver_name() in ['javascript', 'python']:
```

Comment:
Do you happen to remember what was going wrong here or in `test_should_retry_write_until_success_with_leader_change_using_tx_function`?

Reply:
For some reason it did not seem to work, I would appreciate if this could be checked by the relevant authors :)

Reply:
Makes sense :ok_hand: I thought you might have seen something more other than it just failing. Let's leave it at that for now :D

Reply:
Unfortunately, not always there is an obvious reason. Hence, I thought it would be better if it was investigated properly.

Reply:
JS number of retries seemed to be 4 instead of expected 2.
Python just had a timeout.